doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -1,8 +1,15 @@
 
 
-# Decision Trees for Vulnerability Management
+# Decision by Whom about What 
 
```

Comment:
What about this:
Vulnerability Management Decisions

Reply:
Sounds good.